backend/books/views.py

```diff
@@ -53,29 +52,43 @@ def post(self, request):
         }
 
         # Fetch from DB if exist or else get from External DB such as Google Books
-        for isbn in parsed_isbn_list:
-            query_set = Book.objects.filter(isbn_13=isbn)
+        threads = list()
```

Comment:
Just wondering, would it be better to do a single database query first, and then if u want u can multithread the fetching of the data from external database for the ones that are needed? So doing single DB query where u filter by all the ISBNs we are looking for? Just a thought. Probably unneeded because this many queries in quick succession should be nothing at all for our database to handle.

Reply:
Refactored to reflect this